test/base/map.js

```diff
@@ -573,6 +573,14 @@ test("issue 940, should not be possible to change maps outside strict mode", ()
             m.set("x", 1)
```

Comment:
Would be great if we added the tests from #2275, #2255, and missing v5 map tests here as well